src/Microsoft.Azure.WebJobs.Host/Executors/FunctionExecutor.cs

```diff
@@ -133,8 +132,11 @@ public HostOutputMessage HostOutputMessage
                     logCompletedCancellationToken = cancellationToken;
                 }
 
-                await NotifyCompleteAsync(instanceLogEntry, functionCompletedMessage.Arguments, exceptionInfo);
-                _resultsLogger?.LogFunctionResult(instanceLogEntry);
+                if (instanceLogEntry != null)
```

Comment:
Avoid operating on instanceLogEntry if it is null

Reply:
Why is this ever null? That would mean we're missing some important details about the function executing.

Reply:
See above - if NotifyPreBind fails it can be null.

Reply:
What exception does the user see now?

Reply:
"Unable to resolve function name 'xxx'." See the repro test case I added in https://github.com/Azure/azure-functions-host/pull/3094

Reply:
Ok, cool. I just wanted to make sure we weren't running real functions that were having these calls to `Notify***` skipped.